src/test/operationLocationTest.ts

```diff
@@ -0,0 +1,33 @@
+import * as assert from "assert"
+import * as path from "path"
+import * as index from "../index"
```

Comment:
```suggestion
import {OpenApiDiff} from "../index"
```

Reply:
Would be better technically for the others too but this one I think in particular